src/Common/nunit/ConsoleOptions.cs

```diff
@@ -213,10 +213,10 @@ public ConsoleOptions(params string[] args)
 
         // Select tests
 
-        private List<string> inputFiles = new List<string>();
+        private readonly List<string> inputFiles = new List<string>();
```

Comment:
Ok,

Change the standard.

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In src/Common/nunit/ConsoleOptions.cs
> <https://github.com/nunit/nunit/pull/586#discussion_r28155560>:
>
> > @@ -213,10 +213,10 @@ public ConsoleOptions(params string[] args)
> >
> >          // Select tests
> >
> > -        private List<string> inputFiles = new List<string>();
> > +        private readonly List<string> inputFiles = new List<string>();
>
> Not part of our standards but good common sense. Maybe it should be there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28155560>.
>